samples/MvcSandbox/MvcSandbox.xproj

```diff
@@ -17,6 +17,7 @@
     <DnxInvisibleContent Include="bower.json" />
     <DnxInvisibleContent Include=".bowerrc" />
     <DnxInvisibleContent Include="package.json" />
+    <DnxInvisibleContent Include=".npmrc" />
   </ItemGroup>
   <Import Project="$(VSToolsPath)\DNX\Microsoft.DNX.targets" Condition="'$(VSToolsPath)' != ''" />
```

Comment:
Discussed w/ @BillHiebert and we decided that's not worthwhile. For one thing, my script would need to duplicate the logic in `Microsoft.DNX.targets` to choose between `Microsoft.DotNet.Targets` and `Microsoft.DotNet.Web.Targets`. For another, we should eventually switch to `.csproj` files anyway.